dataset-2-1/feature_defs.json

```diff
@@ -1,56 +1,56 @@
-    "description": "The surface area of a cell, as measured by segmented pixels and associated edge length, in units of \u00b5m\u00b2.",
-    "tooltip": "The surface area of a cell, as measured by segmented pixels and associated edge length, in units of \u00b5m\u00b2.",
+    "description": "The surface area of a cell, as measured by the sum of individual outer surface areas of segmented voxels (facing the background) in the segmented cell image, in units of \u00b5m\u00b2.",
+    "tooltip": "The surface area of a cell, as measured by the sum of individual outer surface areas of segmented voxels (facing the background) in the segmented cell image, in units of \u00b5m\u00b2.",
     "unit": "\u00b5m\u00b2",
     "key": "cell-surface-area",
     "discrete": false
   },
   {
     "displayName": "DNA height",
-    "description": "The height of a nucleus, as measured by the number of pixels that separates the bottom most and top most pixel in the segmented image converted to units of \u00b5m.",
-    "tooltip": "The height of a nucleus, as measured by the number of pixels that separates the bottom most and top most pixel in the segmented image converted to units of \u00b5m.",
+    "description": "The height of a nucleus, as measured by the number of voxels that separates the bottom most and top most voxel in the segmented DNA image converted to units of \u00b5m.",
+    "tooltip": "The height of a nucleus, as measured by the number of voxels that separates the bottom most and top most voxel in the segmented DNA image converted to units of \u00b5m.",
     "unit": "\u00b5m",
     "key": "dna-height",
     "discrete": false
   },
   {
     "displayName": "DNA volume",
-    "description": "The volume of a nucleus of a given cell, as measured by segmented voxels and associated edge length, in units of \u00b5m\u00b3.",
-    "tooltip": "The volume of a nucleus of a given cell, as measured by segmented voxels and associated edge length, in units of \u00b5m\u00b3.",
+    "description": "The volume of a nucleus of a given cell, as measured by the sum of individual volumes of segmented voxels in the segmented DNA image, in units of \u00b5m\u00b3.",
+    "tooltip": "The volume of a nucleus of a given cell, as measured by the sum of individual volumes of segmented voxels in the segmented DNA image, in units of \u00b5m\u00b3.",
     "unit": "\u00b5m\u00b3",
     "key": "dna-volume",
     "discrete": false
   },
   {
     "displayName": "DNA surface area",
-    "description": "The surface area of a cell, as measured by segmented pixels and associated edge length, in units of \u00b5m\u00b2.",
-    "tooltip": "The surface area of a cell, as measured by segmented pixels and associated edge length, in units of \u00b5m\u00b2.",
+    "description": "The surface area of a nucleus of a given cell as measured by the sum of individual outer surface areas of segmented voxels (facing the background) in the segmented DNA image, in units of \u00b5m\u00b2.",
```

Comment:
nit(s): this seems like it should be `area of the nucleus of a given cell` as opposed to `a nucleus`. Also the DNA height feature doesn't have the phrase "of a given cell".  And some of the texts use "a given cell" and some others just use "a cell".

Reply:
yeah, I'll changed the 'a' to a 'the'. I agree with the "of a given cell" I remove it for the tooltip.